test/node_roles_functional/background_tasks.config.ts

```diff
@@ -40,7 +40,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         '--usageCollection.usageCounters.bufferDuration=0',
 
         `--plugin-path=${path.resolve(__dirname, 'plugins', 'core_plugin_initializer_context')}`,
-        '--node.roles=["background_tasks"]',
```

Comment:
Could have used `migrations.skip` instead?

Reply:
We need something to create the indices, otherwise other background things like task manager throws unhandled promise rejections and the process exits.